docker-compose.yml

```diff
@@ -51,6 +51,34 @@ services:
       - "${KAVA_CONTAINER_COSMOS_RPC_PORT}"
       - "${KAVA_CONTAINER_EVM_RPC_PORT}"
 
+  # shards are in name only. they are standard peer nodes, but will only recieve traffic
```

Comment:
with `kava-pruning` as the active node in this context?

Reply:
kava-validator is the active node. in the tests, localhost:7777 is the "archive cluster", configured with these backends:
* pruning node -> kava-pruning
* active node -> kava-validator
* shard 2 (blocks 11-20) -> kava-shard-20
* shard 1 (blocks 1-10) -> kava-shard-10

this comment is just referring to the fact that _all_ of the nodes in the test setup have all blocks. their response urls and the request metrics that are created are used to validate that the routing is correct